service/awsconfig/v5/resource/cloudformation/adapter/outputs_test.go

```diff
@@ -3,41 +3,159 @@ package adapter
 import (
 	"testing"
 
-	// NOTE(PK): This import is disturbing. I'm not bothering. It's first candidate to go away.
-	"github.com/giantswarm/aws-operator/service/awsconfig/v5/cloudconfig"
+	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
 )
 
-func TestAdapterOutputsRegularFields(t *testing.T) {
+func Test_CloudFormation_Adapter_Outputs_MasterCloudConfigVersion(t *testing.T) {
```

Comment:
The tests are refactored but ensure the same. There is also an additional test. I am not covering everything, but enough to be comfortable enough for now.